### PR TITLE
use mocha's skip for conditional tests

### PR DESCRIPTION
**Pull request by MichaelMure**
Hopefully I got all of them.

### CODE REVIEW

File: js/src/files-mfs.js [hunk 1]
```diff
@@ -37,10 +37,10 @@ module.exports = (common) => {
     after((done) => common.teardown(done))
 
     describe('.mkdir', function () {
-      it('make directory on root', (done) => {
+      it('make directory on root', function(done) {
```
Comment (diasdavid) [inline js/src/files-mfs.js]: This fails on linting.

Comment (MichaelMure) [reply js/src/files-mfs.js]: ?
The lint pass locally and on travis.
Do you want `function (done)` with a space instead ?

Comment (diasdavid) [reply js/src/files-mfs.js]: @MichaelMure weird, I could bet that standard wouldn't accept that. Yeah I do want the whitespace to make it consistent :)

**Comment by MichaelMure** [comment]
@diasdavid coding style fixed, ready to merge

**Comment by diasdavid** [comment]
Thanks @MichaelMure :)

**Pull request merged**